Week3/Homework/Js-exercises/cardValidator.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const cardValidator = function(cardNumber) {
+  //first converts our number into an array of strings then using for of, converts the string array to numbers array
+  if (Number.isInteger(Number(cardNumber)) !== true) {
+    return console.log("invalid characters!");
+  }
+  let strArr = cardNumber.toString().split("");
+  const numArr = [];
+  for (let ele of strArr) {
+    numArr.push(Number(ele));
+  }
+  console.log(numArr.length);
```

Comment:
You need not push `console.log` in your final code as this feature is only used for debugging purposes. 🙂